docs/firehose.MD

```diff
@@ -132,7 +132,7 @@ For each requested prefix the responder has the option of returning either:
         # There is one 2-tuple for every prefix requested
         (
             # 0 – success
-            # 1 – no data as of the requested block, available blocks should be populated
+            # 1 – no data as of the requested block, the final entry of the response should be non-empty
```

Comment:
The name "available blocks" didn't mean anything to me yet, when reading the example from top to bottom.